src/main/java/Application.java

```diff
@@ -0,0 +1,49 @@
+import controller.GameController;
+import domain.Car;
+import domain.repository.CarRepository;
+import service.GameService;
+import service.validate.InputVerifier;
+import util.Converter;
+import view.input.InputView;
+import view.output.OutputView;
+
+import java.util.List;
+
+public class Application {
+    static GameService gameService = new GameService(new CarRepository());
+    static GameController gameController = new GameController(gameService);
+
+    public static void main(String[] args) {
+        String[] names = getNames();
+        List<Car> cars = makeCars(names);
+        int count = getCount();
+        OutputView.printPhrase();
+        execute(count, cars);
+        OutputView.printResult(cars);
+    }
```

Comment:
main에서 중요한 로직은, 메인로직 실행의 책임이 있는 excute와 화면 노출 책임이 있는 print로 보여요. 

게임을 진행하기 위해 필요한 초기화 작업들은, 메서드로 한번더 감싸줘도 될 것 같아요 :) 

```suggestion
    public static void main(String[] args) {
        initialize(**)
        OutputView.printPhrase();
        execute(count, cars);
        OutputView.printResult(cars);
    }
```

Reply:
검프가 제안해주신 형식이 initialize 함수 안에서 자동차 객체와 시도 횟수 등을 초기화 하고 반환값으로는 cars와  count 이 두 가지를 반환하는 형태가 맞을까요?

Reply:
initialize 함수가 cars와 count를 반환해야할 책임이 있을까요?!
말그대로, 게임을 시작하기위한 초기화작업 ( 특정 저장소에 저장하거나, 랜덤값 세팅을 한다거나 ) 등을 할 수 있을 것 같아요 :)